POC: Increase utilities test coverage (#224)
BBC-archive/psammead: packages/utilities/psammead-test-helpers/CHANGELOG.md

| Version | Description |
|---------|-------------|

> | 0.2.0   | [PR#224](https://github.com/BBC-News/psammead/pull/224) Adds a helper method for testing package export values. Tests the new helper method's error cases. Fianlly, uses the helper method to test it's own exports. |

Reviewer: ```suggestion
| 0.2.0   | [PR#224](https://github.com/BBC-News/psammead/pull/224) Adds a helper method for testing package export values and uses it to test its own exports. |
```
